pages/api/archive.js

```diff
@@ -0,0 +1,20 @@
+import { dbConnect } from '../../src/lib/db/database';
+import Archive from '../../src/lib/models/Archive';
+import Task from '../../src/lib/models/Task';
+
+export default async function handler(req, res) {
+	if (req.method === 'POST') {
+		const oldTaskData = JSON.parse(req.body);
+		await dbConnect();
+		await Archive.insertMany(oldTaskData);
+		const deleteMany = await Task.deleteMany({ done: true });
```

Comment:
It seems like you really update existing data here. A post request should only be used when creating a new resource.

Reply:
Changed to PUT. Thanks